MiniAn/minian_run.py

```diff
@@ -502,35 +503,27 @@
     file_.close()
     
     # Parameters
+    # Set only "Operations" for params_srouce_data
     if "OperationName" in params_source_data:
-        params["Operations"] = params_source_data["OperationName"] + " > " + params["Operations"]
+        if "Operations" not in params_source_data:
+            params_source_data["Operations"] = params_source_data["OperationName"]
+        
         del params_source_data["OperationName"]
-    elif "Operations" in params_source_data:
-        params["Operations"] = params_source_data["Operations"] + " > " + params["Operations"]
-        del params_source_data["Operations"]
-
-    params = {**params, **params_source_data}
-
-    for funcName, funcValue in advanced_settings.items():
-        if type(funcValue) is dict:
-            for variableName, variableValue in funcValue.items():
-                params["Advanced: "+funcName+" > "+variableName ] = str(variableValue) if type(variableValue) is not str else '"'+variableValue+'"'
 
-    if "BinningFactor" in params:
-        params["BinningFactor"] *= spatial_downsample
-    elif spatial_downsample > 1 :
-        params["BinningFactor"] = spatial_downsample
+    if spatial_downsample > 1:
+        params_doric["BinningFactor"] = spatial_downsample
```

Comment:
Because I just added it back today after seeing your comment in ImageController.

Also before I was merging the spatial_downsample with the BinningFactor of the binning operation. Because I removed everything I added the specific case for mininan in ImageController

Reply:
Oups, I see. Thanks!